contracts/handlers/CentrifugeAssetHandler.sol

```diff
@@ -66,7 +66,7 @@ contract CentrifugeAssetHandler is IDepositHandler {
             metaDataHash := mload(add(data, 0x20))
         }
 
-        require(_assetDepositStatuses[metaDataHash] == false, "asset hasn't been deposited or has already been finalized");
+        require(_assetDepositStatuses[metaDataHash] == false, "asset has been deposited or has already been finalized");
```

Comment:
You could remove "...or has already been finalized" since the status enum was removed